docs/decisions/0002-name-affirmation-plugin.rst

```diff
@@ -0,0 +1,33 @@
+2. Proctoring Email Template Overrides
+--------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+`edx-name-affirmation` is intended to be an optional feature for the open edX community. As such, we
+should allow the consuming application (in our case, `edx-platform <https://github.com/edx/edx-platform>`_)
+to determine whether or not this feature should be installed.
+
+Decision
```

Comment:
Could we place have a section called `options`. Please list out all the different design options you hashed out. See the list of options you documented at https://openedx.atlassian.net/wiki/spaces/PT/pages/2594570529/Discovery+Name+Affirmation+Technical+Design